homework.js

```diff
@@ -2,25 +2,49 @@
 // declare a variable called fruit
 // assign your variable the value 6
 // console log your variable
+let fruit = 6;
+console.log(fruit)
 
-
-// write a function called product that accepts two numbers and 
+// Problem 2 write a function called product that accepts two numbers and 
 // logs the product in the console and then
 // returns the answer when you multiply them both together
 // call that function on the numbers 4 & 689 to test it
-
-// Problem 2
+function product(num1, num2) {
+    let(num1 * num2);
+    return product;
+}
+console.log(4 * 689)
+// Problem 3
 // write a function called evens that takes an array of numbers, removes 
 // all the numbers that are odd and returns a new array with only the even numbers
 // remember that even numbers have a remainder of 0 when you divide them by 2
 // call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
-
-//Problem 3
+function evens(arr) {
+    let result = arr.filter(arr => arr % 2 == 0);
+    return result;
+}
+console.log(evens([4, 37, 18, 52, 103, 22]));
+}
+//Problem 4
 // write a function called five that takes a word and prints it in the console
 // five times
 // call your function on the word "pineapple" to test it
-
-//Problem 4
+for (let counter = 0; counter < 5; counter++) {
+    console.log("pineapple")
+}
+//Problem 5
 // write a function called shortWords that takes an array of words and then
 // returns a new array with only the words that are 3 letters long or less
 // call your function on this array ["basin", "pit", "cheese", "bun", "river"]
+function shortWords ["basin", "pit", "cheese", "bun", "river"];
+let length = array.length[3];
+// let max = 3;
+// for (let length of array)
+for (i = 0; i < shortWords.length; i++) {
+    if (length - newArray);
+}   then (return shortWords);
```

Comment:
Return ends the function, so you only want to do that once, not inside a loop. In this kind of loop, use `i` for an index to stop at the index of each word one at a time.
`if (shortwords[i].length <= 3){
    newArray.push(shortwords[I]`
or something like that.